lib/hipchat/api_version.rb

```diff
@@ -302,6 +302,15 @@ def view_config
         }[version]
       end
 
+      def user_room_config
```

Comment:
I think the name is misleading. How about 'user_joined_rooms' or something similar?